github.go

```diff
@@ -219,8 +223,20 @@ func callGitHubApi(repo GitHubRepo, path string, customHeaders map[string]string
 		request.Header.Set(headerName, headerValue)
 	}
 
-	resp, err := httpClient.Do(request)
+	// This looks slightly artificial, but if retries = 0, we never invoke the loop,
+	// but intuitively on the CLI retries is the number of times we _retry_ as opposed
+	// to try. There are alternative ways of structuring it, but we should probably
+	// just replace this with retryablehttp anyway.
 
+	for retries +=1; retries > 0; retries -= 1 {
+		resp, err := httpClient.Do(request)
+		if err != nil {
+			fmt.Printf("Error encountered downloading from %s: %s; continuing", request, err)
+			time.Sleep(retrySleepSeconds * time.Second) // Effective linear backoff
+		}
```

Comment:
Same issue as above re: missing break. Also, perhaps we can refactor this into a function `HttpDoWithRetry`?

Reply:
Done.